test/Google.spec.js

```diff
@@ -0,0 +1,10 @@
+describe('Given a SDET learning protractor', () => {
+  describe('when open Google Page', () => {
+    beforeEach(() => {
+      browser.get('http://www.google.com');
+    });
```

Comment:
para mayor legibilidad separa los steps (describe, before, it, etc, etc) con una linea en blanco